docs/pages/management/admin/labels.mdx

```diff
@@ -96,6 +96,17 @@ where you installed the Teleport agent.
        environment: dev
    ```
 
+   This example illustrate a simple value setting. However, you can use static labels
```

Comment:
I'm a little confused by this. It reads to me as if you're going to illustrate a simple example that does not include "more complex string values that include white space" but then the example does just that.

Reply:
I was surprised that it worked with white space and punctuation marks, so it seemed like a cool thing to point out.